$OpenBSD: patch-appdefs_trig,v 1.1 2010/12/06 12:08:55 stephan Exp $

# packaged appdefs.trig db is out of date, patch it using a newer
# one provided at http://freeworld.thc.org/thc-amap/appdefs.trig

--- appdefs.trig.orig	Mon Dec  6 12:20:49 2010
+++ appdefs.trig	Mon Dec  6 12:21:00 2010
@@ -1,4 +1,4 @@
-###V:2###DO NOT EDIT THIS LINE!
+###V:6###DO NOT EDIT THIS LINE!
 #
 # This is the trigger file "appdefs.trig" for amap
 #
@@ -45,7 +45,8 @@
 #
 
 # neither change name or value of this one
-http:80,81,82,8000,8080,8081,8888:tcp:0:"GET / HTTP/1.0\r\n\r\n"
+http-get:80,81,82,8000,8080,8081,8888:tcp:0:"GET / HTTP/1.0\r\n\r\n"
+http-head:80,81,82,8000,8080,8081,8888:tcp:0:"HEAD / HTTP/1.0\r\n\r\n"
 
 #
 # ENTER YOUR RESPONSES HERE
@@ -56,6 +57,8 @@ http:80,81,82,8000,8080,8081,8888:tcp:0:"GET / HTTP/1.
 #
 # CURRENT TRIGGER DATABASE
 #
+http-proxy-ident:80,81,82,8000,8080,8081,8888:tcp:0:"TRACE HTTP://localhost HTTP/1.0\r\n\r\n"
+http-trace:80,81,82,8000,8080,8081,8888:tcp:0:"TRACE / HTTP/1.0\r\n\r\n"
 ssl:443:tcp:1:0x80 80 01 03 01 00 57 00 00 00 20 00 00 16 00 00 13 00 00 0a 07 00 c0 00 00 66 00 00 07 00 00 05 00 00 04 05 00 80 03 00 80 01 00 80 08 00 80 00 00 65 00 00 64 00 00 63 00 00 62 00 00 61 00 00 60 00 00 15 00 00 12 00 00 09 06 00 40 00 00 14 00 00 11 00 00 08 00 00 06 00 00 03 04 00 80 02 00 80 63 b9 b9 19 c0 2b ae 90 74 4c 73 eb 8b cf d8 55 ea d0 69 82 1b ef 23 c3 39 9b 8e b2 49 3c 5a 79
 ms-remote-desktop-protocol:3389:tcp:1:0x03 00 00 0b 06 e0 00 00 00 00 00
 #ms-remote-desktop-protocol:3389:tcp:1:0x03 00 00 0b 06 e0 00 00 00 00 00 43 6f 6f 6b 69 65 3a 20 6d 73 68 3d 61 6d 61 70 0d 0a
@@ -79,12 +82,14 @@ snmp-public:161:udp:1:0x30 82 00 2f 02 01 00 04 06 70 
 ms-sql::udp:1:0x02
 jrmi::tcp:1:0x4a 52 4d 49 00 02 4b
 nessus:1241:tcp:0:"< NTP/1.2 >\n"
-webmin:10000:tcp:0:0x47 45 54 20 2f 20 48 54 54 50 2f 31 2e 30 0d 0a 0d 0a
 netop:6502:udp:0:0xd6 81 81 52 00 00 00 f3 87 4e 01 02 32 00 a8 c0 00 00 01 13 c1 d9 04 dd 03 7d 00 00 0d 00 54 48 43 54 48 43 54 48 43 54 48 43 54 48 43 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 02 32 00 a8 c0 00 00 00 00 00 00 0 0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
 db2::tcp:1:0x00 35 00 00 01 00 00 00 29 00 00 00 53 51 4c 44 42 32 52 41 01 00 00 00 00 00 07 00 01 00 02 00 01 00 00 00 01 00 00 80 00 00 00 01 04 00 00 00 01 00 00 80 01
 jinilookupservice::tcp:0:0x00 00 00 01
 slp:427:tcp:0:0x02 03 00 00 4b 00 00 00 00 00 00 00 00 02 65 6e 00 ff ff 00 1c 73 65 72 76 69 63 65 3a 74 68 63 3a 2f 2f 31 32 37 2e 30 2e 30 2e 31 3a 31 33 33 37 00 00 0b 73 65 72 76 69 63 65 3a 74 68 63 00 07 64 65 66 61 75 6c 74 00 00 00
 tivoli_tsm-server:1500:tcp:0:0x00 04 1d a5
 norman-njeeves:2868:tcp:0:0x11
+edonkey-client:4662:tcp:0:0xe3 3d 00 00 00 01 10 54 48 43 54 48 43 54 48 43 54 48 43 54 48 43 21 7f 00 00 01 36 12 04 00 00 00 97 01 65 53 65 72 76 65 72 89 11 3c 83 fb 00 ef 06 00 97 55 65 73 65 72 76 65 72 00 00 00 00 00 00
+lotus-notes:1352:tcp:0:0x3a 00 00 00 2f 00 00 00 02 00 00 40 02 0f 00 01 00 3d 05 00 00 00 00 00 00 00 00 00 00 00 00 2f 00 00 00 00 00 00 00 00 00 40 1f 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+dtspcd:6112:tcp:0:0x33 31 33 37 30 30 30 32 30 34 30 30 30 64 30 30 30 31 20 20 34 20 00 72 6f 6f 74 00 00 43 4a 00 00
 #old:
 #oracle-tns-listener:1521:tcp:1:0x01 0c 00 00 01 00 00 00 01 36 01 2c 00 00 08 00 7f ff a3 0a 00 00 01 00 00 d2 00 3a 00 00 02 00 61 61 00 00 00 00 00 00 00 00 00 00 06 18 00 01 32 80 00 00 00 00 00 00 00 00 28 44 45 53 43 52 49 50 54 49 4f 4e 3d 28 41 44 44 52 45 53 53 3d 28 50 52 4f 54 4f 43 4f 4c 3d 54 43 50 29 28 48 4f 53 54 3d 31 31 31 2e 31 31 2e 31 31 2e 31 31 29 28 50 4f 52 54 3d 31 35 32 31 29 29 28 43 4f 4e 4e 45 43 54 5f 44 41 54 41 3d 28 53 45 52 56 49 43 45 5f 4e 41 4d 45 3d 68 69 68 69 29 28 53 52 56 52 3d 44 45 44 49 43 41 54 45 44 29 28 43 49 44 3d 28 50 52 4f 47 52 41 4d 3d 43 3a 5c 50 72 6f 67 72 61 6d 20 46 69 6c 65 73 5c 4f 72 61 63 6c 65 5c 6a 72 65 5c 31 2e 31 2e 37 5c 62 69 6e 5c 6a 72 65 77 2e 65 78 65 29 28 48 4f 53 54 3d 41 41 41 41 41 41 41 41 41 29 28 55 53 45 52 3d 41 41 41 41 41 41 41 29 29 29 29
